Makefile

```diff
@@ -72,6 +72,11 @@ menuconfig: Kconfig
 	MENUCONFIG_STYLE=aquatic \
 	$(KCONFIG_MCONF) $<
 
+# Generate config for the current architecture (only)
```

Comment:
Shouldn't we build arm64 and amd64 by default?

Reply:
Possibly. But we build for _everything_ by default. This was a request for something between "minimal" and "all"

> What I have wanted in the past is something like `make minimal templates install`, so I still get all the templates, but only the native guestagent. Because minimal only installs the default template.

Going with files (instead of a script) is probably easiest for discussion, even if config is somewhat duplicated.